source/includes/steps-start-sharded-cluster.yaml

```diff
@@ -0,0 +1,48 @@
+title: "Start config servers."
+stepnum: 1
+ref: start-config-servers
+content: |
+
+   Start the :binary:`~bin.mongod` instance for each of the
+   :doc:`config servers </core/sharded-cluster-config-servers>`, specifying
+   ``--configsvr``, ``--replSet``, ``--bind_ip``, and other options as
+   appropriate to your deployment.
+
+   .. code-block:: shell
+
+      mongod --configsvr --replSet <replicaSetName> --bind_ip localhost,<hostname(s)|ip address(es)>
+
```

Comment:
I wonder if we should add some sort of 'check' state here, e.g. 'Connect to the primary mongod and run rs.status() to confirm the health and availability of each CSRS member'.

Reply:
done